neo4j/datadog_checks/neo4j/neo4j.py

```diff
@@ -354,4 +354,7 @@ def get_whitelisted_metrics():
             'db_query_execution_success_total': 'db.query.execution.success.total',
             'db_query_execution_failure_total': 'db.query.execution.failure.total',
             'db_query_execution_latency_millis': 'db.query.execution.latency.millis',
+
+            # GDS metrics
+            'gds_graphs_created': 'gds.graphs_created',
```

Comment:
In the running version on AuraDS we still had the db name inside the metrics.
Will change this